cranelift/codegen/Cargo.toml

```diff
@@ -25,16 +25,18 @@ gimli = { version = "0.21.0", default-features = false, features = ["write"], op
 smallvec = { version = "1.0.0" }
 thiserror = "1.0.4"
 byteorder = { version = "1.3.2", default-features = false }
-peepmatic-runtime = { path = "../peepmatic/crates/runtime", optional = true, version = "0.2.0" }
+peepmatic = { path = "../peepmatic", optional = true, version = "0.65.0" }
+peepmatic-traits = { path = "../peepmatic/crates/traits", version = "0.65.0" }
```

Comment:
Should this dependency also have `optional = true`, so that it isn't pulled in if peepmatic isn't enabled?

Reply:
Good catch!